### PR TITLE
Add number of tax units with non-positive tax liabilities to diagnostic table

### PR DESCRIPTION
**Pull request by andersonfrailey**
This PR addresses issue web-app issue [#344](https://github.com/OpenSourcePolicyCenter/webapp-public/issues/344#issuecomment-276152252). It adds rows to the diagnostic tables produced in `utils.py` that show how many tax-units would have a zero or negative tax liability in a given year. There is no change to any of the output values, but `pufcsv_agg_expect.txt` was changed to add these two additional rows.

### CODE REVIEW

File: taxcalc/tests/pufcsv_agg_expect.txt [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+Tax before Credits ($b)                  1,389.7  1,565.7  1,658.1  1,742.1  1,815.1  1,876.6  1,933.7  1,993.9  2,072.1  2,162.2
+Refundable Credits ($b)                     99.5     97.6     97.7     96.8     96.4     98.2    100.3    102.1    104.0    105.8
+Nonrefundable Credits ($b)                  72.9     73.4     73.7     73.9     74.7     75.1     75.5     76.1     76.7     77.8
+Reform Surtaxes ($b)                         0.0      0.0      0.0      0.0      0.0      0.0      0.0      0.0      0.0      0.0
+Ind Income Tax ($b)                      1,246.4  1,433.1  1,530.6  1,618.5  1,691.9  1,751.7  1,806.8  1,866.0  1,943.2  2,032.6
+Payroll Taxes ($b)                         924.4    958.3    999.5  1,039.7  1,093.5  1,140.7  1,184.3  1,229.9  1,278.4  1,329.9
+Combined Liability ($b)                  2,170.8  2,391.3  2,530.1  2,658.2  2,785.4  2,892.4  2,991.1  3,095.9  3,221.7  3,362.5
+Non-Positive Income Tax Liability (#m)      70.8     70.3     70.4     70.5     70.6     71.5     72.5     72.9     73.6     74.3
+Non-Positive Payroll Tax Liability (#m)     36.8     38.0     38.9     40.1     41.2     42.4     43.7     44.4     45.5     46.8
```
Comment (MattHJensen) [inline taxcalc/tests/pufcsv_agg_expect.txt]: Let's do "Non-Positive Combined Tax Liability" instead of non-positive payroll tax liability..

Comment (andersonfrailey) [inline taxcalc/tests/pufcsv_agg_expect.txt]: Fixed in latest push

**Comment by martinholmer** [comment]
@andersonfrailey, your #1170 pull request is just fine with respect to the logic, but the diagnostic table in `pufcsv_agg_expect.txt` is now virtually unreadable because your new row names are so long.  Please shorten the new row names, so that the table is still 80 characters wide.

**Comment by andersonfrailey** [comment]
@martinholmer latest commit should fix that issue.

**Comment by martinholmer** [comment]
@andersonfrailey, The new row names are much better from a format point of view, but are they accurate?

Now you have these row names:
```
Neg. Income Tax Liability (#m)
Neg. Combined Liability (#m)
```
But they are really the number with non-positive liability.
How about something like this:
```
With Income Tax <= 0 (#m)
With Payroll Tax <= 0 (#m)
```

**Comment by andersonfrailey** [comment]
@martinholmer I like your suggestions. Though I changed the second one to `With Combined Tax <= 0 (#m)`.

**Comment by martinholmer** [comment]
@andersonfrailey said:
> I like your suggestions. Though I changed the second one to `With Combined Tax <= 0 (#m)`.

Of course, thanks for correcting my mistake: the second new line is for combined taxes, not payroll taxes.

**Comment by martinholmer** [comment]
Looks good now; thanks @andersonfrailey.  Merging now.

**Pull request merged**